Source/Libraries/FaultData/DataWriters/Emails/EventEmailService.cs

```diff
@@ -387,7 +386,10 @@ public void ApplyChartTransform(List<Attachment> attachments, XDocument htmlDocu
                     string chartEventID = (string)element.Attribute("eventID") ?? "-1";
                     string cid = $"event{chartEventID}_chart{index:00}.png";
 
-                    Stream image = ChartGenerator.ConvertToChartImageStream(connection, element);
+                    string ignoreMin = ((string)element.Attribute("ignoreMin") ?? "false");
+                    int passedMinimum = bool.Parse(ignoreMin) ? -1 : minimumSamples;
```

Comment:
A name like `noUpsample` would probably be much clearer than `ignoreMin`, but even better than that would be an option to specify a different upsampling threshold to override the global setting.

Reply:
Please see newest commit, it should implement what you suggested.